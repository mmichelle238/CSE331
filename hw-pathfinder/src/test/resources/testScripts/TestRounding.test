# Test to see if the values of the edges become rounded
# and if the total cost also becomes rounded.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 2.3347
AddEdge simpleGraph A C 1.0465
AddEdge simpleGraph C D 4.6577
AddEdge simpleGraph D B 1.1235

FindPath simpleGraph A D