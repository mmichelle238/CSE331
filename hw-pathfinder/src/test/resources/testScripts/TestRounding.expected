# Test to see if the values of the edges become rounded
# and if the total cost also becomes rounded.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 2.335 from A to B in simpleGraph
added edge 1.047 from A to C in simpleGraph
added edge 4.658 from C to D in simpleGraph
added edge 1.124 from D to B in simpleGraph

path from A to D:
A to C with weight 1.047
C to D with weight 4.658
total cost: 5.704